qhub/cost.py

```diff
@@ -0,0 +1,112 @@
+import json
+import logging
+import os
+import re
+import subprocess
+
+from rich.console import Console
+from rich.table import Table
+
+logger = logging.getLogger(__name__)
+
+
+def _check_infracost():
+    """
+    Check if infracost is installed
+    """
+    try:
+        subprocess.check_output(["infracost", "--version"])
+        return True
+    except subprocess.CalledProcessError:
+        return False
+
+
+def _check_infracost_api_key():
+    """
+    Check if infracost API key is configured
+    """
+    try:
+        subprocess.check_output(["infracost", "configure", "get", "api_key"])
+        return True
+    except subprocess.CalledProcessError:
+        return False
+
+
+def _run_infracost(path):
+    """
+    Run infracost on the given path and return the JSON output
+    """
+    try:
+        process = subprocess.Popen(
+            ["infracost", "breakdown", "--path", path, "--format", "json"],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+        )
+        stdout, stderr = process.communicate()
+        infracost_data = json.loads(
+            re.search("({.+})", stdout.decode("UTF-8"))
+            .group(0)
+            .replace("u'", '"')
+            .replace("'", '"')
+        )
+        return infracost_data
+    except subprocess.CalledProcessError:
+        return None
+
+
+def infracost_report(path):
+    """
+    Generate a report of the infracost cost of the given path
+    args:
+        path: path to the qhub stages directory
+    """
+    if not path:
+        path = os.path.join(os.getcwd(), "stages")
+
+    if _check_infracost() and _check_infracost_api_key():
+        if not os.path.exists(path):
+            print("Deployment is not available")
+        else:
+            data = _run_infracost(path)
+
+            cost_table = Table(title="Cost Breakdown")
+            cost_table.add_column("Name", justify="right", style="cyan", no_wrap=True)
+            cost_table.add_column("Cost", justify="right", style="cyan", no_wrap=True)
```

Comment:
`Cost ($)` perhaps?

Reply:
Or can we change the currency value?

Reply:
@iameskild We can change currencies! However, do we want to prioritise the feature right now?

Reply:
I think just adding `(USD)` is a good solution

Reply:
I agree, adding a way to change currencies is not a priority but let's add a label. `USD` as Kim suggested sounds good to me :)

@HarshCasper, would you mind just creating an issue listing a few of the enhancement requests raised in this PR and any others that you might suggest? Thanks a lot!!